modules/analyse/src/main/Advice.scala

```diff
@@ -110,6 +113,7 @@ private[analyse] object MateAdvice {
         case MateLost if nextCp > 999 => Inaccuracy
         case MateLost if nextCp > 700 => Mistake
         case MateLost => Blunder
+        case MateDelayed => Neutral
```

Comment:
The compiler forbade `case MateDelayed if false => Inaccuracy` so I guess this is the next-best possibility?

Reply:
I don't like these new glyph and "neutral" object, they don't make sense here. But I see this `apply` function returns an `Option[MateAdvice]` so returning a `None` is an option, where `val judgment` is an Option itself

Reply:
I tried wrapping everything in `Option` and the code became a mess, however I think I found a cleaner solution.  (But possibly cleanest is removing `MateDelayed` entirely?)